src/helpers/misc.ts

```diff
@@ -529,3 +529,16 @@ export function moveItemToIndex<T>(array: T[], startIndex: number, targetIndex:
   array.splice(targetIndex, 0, item);
   return array;
 }
+
+/**
+ * Determine if the numbers in a set are consecutive.
+ */
+export function isConsecutive(set: Set<number>): boolean {
```

Comment:
```suggestion
export function isConsecutive(set: Iterable<number>): boolean {
```
so we can use this in the future without bothering about creating a set